parcels/fieldset.py

```diff
@@ -1119,11 +1121,14 @@ def computeTimeChunk(self, time, dt):
                                 f.data[2, :] = None
                         f.data[1:, :] = f.data[:2, :]
                         f.data[0, :] = data
-                g.load_chunk = np.where(g.load_chunk == 3, 0, g.load_chunk)
+                g.load_chunk = np.where(g.load_chunk == g.chunk_loaded_touched_in_JIT,
+                                        g.chunk_loading_requested, g.load_chunk)
```

Comment:
well, this line is redundant - the data are still used by JIT, they are already loaded, so they do not need to be loaded again. This would just request unnecessary I/O for data already in memory.

Please comment / remove the line and test if it still works (which it should).

Reply:
Not true. This line was necessary to fix a bug in #978; took me a long time to find, but when a new FileBuffer instance is created, in some cases the chunking otherwise goes wrong. So I'm pretty sure now that we need this line